notificationworkerlambda/src/main/scala/com/gu/notifications/worker/delivery/DeliveryService.scala

```diff
@@ -92,11 +75,8 @@ class DeliveryServiceImpl[F[_], C <: DeliveryClient] (
 
     for {
       payload <- Stream.eval(payloadF)
-      res <- tokens
-        .map(token => sending(client)(token, payload))
-        .parJoin(maxConcurrency)
+      res <- sending(client)(token, payload)
     } yield res
   }
 
-  def close(): F[Unit] = F.delay(client.close())
```

Comment:
Was that unused?

Reply:
I belive so, I expected a compiler error when I deleted that line and it didn't happen.
 It might be valuable to re-use connections as much as possible (especially over a nat) so this may be a good thing (for now) and given the Lambda lifecycle that might mean not closing between invocations.
Let's chat about this too as either way the code is a bit confusing with close's referenced in the service but not used.